wurst/_handles/DialogBox.wurst

```diff
@@ -0,0 +1,67 @@
+package DialogBox
+import HashMap
+import Dialog
+import LinkedList
+
+/**DialogBox lets you to create dialogs and associate the buttons with code.*/
+public class DialogBox
+
+    private static constant HashMap<button, trigger> buttonTriggers = new HashMap<button, trigger>
+    private static constant HashMap<dialog, DialogBox> dialogBoxes = new HashMap<dialog, DialogBox>
+    private constant LinkedList<button> buttons = new LinkedList<button>
+
+    private constant dialog dialogHandle = createDialog()
+    private constant trigger clickDialog = CreateTrigger()
+        ..registerDialogEvent(this.dialogHandle)
+        ..addAction(function removeClickedDialog)
```

Comment:
I only ment the `Create` part.
Move 
```
..registerDialogEvent(this.dialogHandle)
..addAction(function removeClickedDialog)
```

back into constructor.
Change `..addAction(function removeClickedDialog)` to `..addAction(() -> removeClickedDialog())`

Reply:
Oh ok, I was already wondering, if executing functions there makes sense.

What is the benefit of using () -> compared to function in this context?